CHANGES.md

```diff
@@ -1,10 +1,19 @@
-XXXXX (202XXXXX)
+0.4 (2021-XX-XX)
 ================
 
 Breaking changes:
 
 -   The protocol parsers now use the cereal library and the Pinch.Protocol.Parser module has been removed.
-    Custom protocol implementations need to be ported to the cereal parsers.
+    Custom protocol implementations need to be ported to the cereal parsers. ( #28 )
+-   Use new header format for binary protocol. This fixes some compatibility issues with thrift libraries
+    for other languages. ( #33)
+
+Other changes:
+
+-   Add implementations of the framed/unframed transports ( `Pinch.Transport`, #29 ).
+-   Add Thrift RPC client/server functionality ( `Pinch.Server` / `Pinch.Client` )
+-   A new code generator, `pinch-gen`, is available. It can generate code from Thrift files for use
+    with the pinch library. For more details, see  https://github.com/phile314/pinch-gen/ .
```

Comment:
```suggestion
-   Add implementations of the framed/unframed transports (`Pinch.Transport`, #29).
-   Add Thrift RPC client/server functionality (`Pinch.Server`/`Pinch.Client`).
-   A new code generator, `pinch-gen`, is available. It can generate code from Thrift files for use
    with the pinch library. For more details, see https://github.com/phile314/pinch-gen/.
```